tgui/packages/tgui/interfaces/LogViewer.tsx

```diff
@@ -281,9 +269,9 @@ const LogViewerContent = (props: LogViewerContentProps, context: any) => {
   const { entries, extendedTarget, setExtendedTarget, ...rest } = props;
   return (
     <Section title={entries.length + ' Entries'} {...rest}>
-      <Flex direction="column">
+      <Stack vertical grow>
         {entries.map((entry) => (
-          <Flex.Item key={entry.key}>
+          <Stack.Item key={entry.key}>
```

Comment:
You can replace this Stack.Item with a Box and have a similar effect (minus the gaps). The Stack itself can be removed.